styleguide/derek/templates/landing-page/landingpage-tradeshow-event.ejs

```diff
@@ -0,0 +1,104 @@
+<div class="rsLpEvent-announcementRiver">
+  <div class="rsLpEvent-announcementRiverContainer">
+    <div class="rsLpEvent-announcementRiverRow">
+      <div class="rsLpEvent-announcementRiverTitle">
+        <h3 class="rsLpEvent-announcementRiverTitleHeader">
+          Network with local IT leaders and Rackspace SMEs. Discuss how
+          to drive business results through continuous IT modernization
+          over dinner.
+        </h3>
+      </div>
+      <div class="rsLpEvent-announcementRiverDetails">
+        <div class="rsLpEvent-announcement-col">
+          <span class="rsLpEvent-announcementRiver-secondary">
+            Matsuhisa<br/>
+            98 Steele St<br/>
+            Denver, CO<br/>
+            80206<br/>
+          </span>
+        </div>
+        <div class="rsLpEvent-announcement-col">
+          <span class="rsLpEvent-announcementRiver-secondary rsLpEvent-announcementRiver-date">
+            Thursday, April 11<br/>
+            6 - 8:30 PM
+          </span>
```

Comment:
Would you mind using the dummy content Mike had in his mockup?  
![image](https://user-images.githubusercontent.com/5263371/55434194-9c59e500-555c-11e9-8e91-4cb6dd255cd4.png)

The header in your version is really long, so I wanna make sure it's okay with the shorter content as well (which it should be).